assets/presto_clp/Dockerfile

```
@@ -0,0 +1,62 @@
+    tmux \
+    unzip \
+    vim \
+    wget \
+    zip \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*;
+
+# Install Task
+RUN cd /usr/local && sh -c "$(curl --location https://taskfile.dev/install.sh)" -- -d;
+
+RUN pip3 install pyarrow;
```

Hadolint (2.12.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)